ocaml/idl/datamodel.ml

```diff
@@ -857,6 +859,8 @@ let _ =
     ~doc:"The hosts in this pool are not compatible." ();
   error Api_errors.pool_hosts_not_homogeneous [ "reason" ]
     ~doc:"The hosts in this pool are not homogeneous." ();
+  error Api_errors.pool_management_cannot_be_set_on_system_network [ "network" ]
+    ~doc:"You tried to configure pool management on system network: It is not allowed." ();
```

Comment:
Based on looking at the code to work out the circumstances in which the error is raised, this should be
"You tried to configure pool management to use a system network. This is not allowed."